tests/UKHO.ERPFacade.API.FunctionalTests/Helpers/SAPXmlHelper.cs

```diff
@@ -613,7 +665,7 @@ private static bool VerifyCreateENCCell(string childCell, ZMAT_ACTIONITEMS item,
 
         private static bool VerifyDecryptedPermit(string childCell, ZMAT_ACTIONITEMS item, string permitState)
```

Comment:
As console line is commented in this method so childCell parameter looks not used any more, can we remove this?

Reply:
done